pkgs/tools/virtualization/mkosi/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, pkgs
+, fetchFromGitHub
+, python3
```

Comment:
similar to the above. if you use the callPackage under python 3, you can use the python libs directly as inputs.

Reply:
Hey, thanks for your review. I'm not quite sure what you are suggesting here. Are you suggesting to callPackage this in `pkgs/top-level/python-packages.nix` ? Mkosi isn't a Python library, only a binary.

Reply:
no, use `python3Packages.callPackage` in `all-packages.nix`, and then replace the input `python3` with the actual libs

Reply:
Done. Thanks for explaining. :)